### PR TITLE
[now-client] Add `apiUrl` support

### PR DESCRIPTION
**Pull request by rdev**
This PR adds support for `apiUrl` option to `now-client` as well as a test for it

### CODE REVIEW

File: packages/now-cli/test/integration.js [hunk 1]
```diff
@@ -529,6 +529,43 @@ testv1('deploy a v1 dockerfile project', async t => {
...
+
+  const { stdout, stderr, code } = await execa(
+    binaryPath,
+    [
+      directory,
+      '--public',
+      '--name',
+      session,
+      '--api',
+      'https://zeit.co/api',
```
Comment (styfle) [inline packages/now-cli/test/integration.js]: I think you need to add this flag to the CLI Help output too

File: packages/now-cli/test/integration.js [hunk 2]
```diff
@@ -529,6 +529,43 @@ testv1('deploy a v1 dockerfile project', async t => {
   context.deployment = host;
 });
 
+test('deploy with a custon API URL', async t => {
```
Comment (styfle) [inline packages/now-cli/test/integration.js]:
```suggestion
test('deploy with a custom API URL', async t => {
```

File: packages/now-cli/src/util/deploy/process-deployment.ts [hunk 3]
```diff
@@ -139,6 +140,7 @@ export default async function processDeployment({
           deploySpinner();
         }
 
+        console.log(event.payload, event.payload.stack);
```
Comment (TooTallNate) [inline packages/now-cli/src/util/deploy/process-deployment.ts]: Remove

File: packages/now-cli/test/integration.js (hunk 1 above)
Comment (styfle) [reply packages/now-cli/test/integration.js]: Nevermind, I see there is already a `--api` flag in the CLI.

**Review by styfle — APPROVED** [review]
Looks good once the tests pass, thanks!

**Pull request merged**